docker-compose.yml

```diff
@@ -4,7 +4,7 @@ services:
   # TODO: change this name to match the name of your app
   portal:
     build:
-      context: .
+      context: ..
```

Comment:
typo?

Reply:
Yes! Thank you I would have never found that!